tests/Unit/API/Site/Controllers/Ads/AccountControllerTest.php

```diff
@@ -216,6 +217,20 @@ public function test_get_billing_status() {
 		$this->assertEquals( 200, $response->get_status() );
 	}
 
+	public function test_get_ads_account_has_access() {
```

Comment:
Came we rename this function or add a docblock to describe what this test is expected to do. It's testing "does not have access", but that's not implied by the function name.

Reply:
Handled in 40148eb.